datahub-ranger-plugin/src/main/java/com/datahub/authorizer/plugin/ranger/DatahubRangerAuthPlugin.java

```diff
@@ -0,0 +1,59 @@
+package com.datahub.authorizer.plugin.ranger;
+
+import org.apache.ranger.plugin.client.BaseClient;
+import org.apache.log4j.Logger;
+import org.apache.ranger.plugin.service.RangerBaseService;
+import org.apache.ranger.plugin.service.ResourceLookupContext;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Datahub Apache Ranger Plugin.
+ * It assists in creating policies on Apache Ranger Admin Portal.
+ *
+ */
+public class DatahubRangerAuthPlugin extends RangerBaseService {
```

Comment:
Minor: We try to use DataHub instead of Datahub in our names.. 

Please rename this to DataHubRangerAuthPlugin..